vaccine_feed_ingest/runners/us/carbon_health/normalize.py

```diff
@@ -0,0 +1,301 @@
+#!/usr/bin/env python
+
+import datetime
+import os
+import pathlib
+import re
+import sys
+from typing import Any, List, Optional
+
+import orjson
+import us
+from vaccine_feed_ingest_schema import location
+
+from vaccine_feed_ingest.utils.log import getLogger
+from vaccine_feed_ingest.utils.normalize import normalize_zip
+
+# Performance optimization.
+#
+# This is different from the usual python-based normalize stages in this repo.
+# The difference is justified because it saves a lot of time[1] by disabling
+# validation.  Disabling this validation is ok because the ingestion framework
+# already does exactly the same validation after calling this code.
+#
+# You can get the normal behavior by flipping `optimized` to False.
+#
+# [1]. In my tests, most of the time spent running this normalizer was spent
+# running validations. Flipping optimized to True cut the running time in half.
+# This normalizer has a larger dataset than average, so perf wins are more
+# significant.
+optimized = True
+if optimized:
```

Comment:
Pydantic has a method for creating a model without validation. I think it would be simplier to use that https://pydantic-docs.helpmanual.io/usage/models/#creating-models-without-validation

Reply:
Oh, I didn't know about that. Thanks for the pointer.

I'd like to advocate for keeping the code as-is, though. (Of course the decision is up to you.) The biggest reason is that the current code is still notably faster even than using `construct`. I just [tried it out](https://github.com/jeremyschlatter/vaccine-feed-ingest/commit/89ae01842679b37933fd4f9f77ae912af41e1d12), and the current code beats the `construct`-based version 11s to 17s. I also like that the current version only requires this one change at the top, rather than having to change every individual constructor call.

lmk what you think.

Reply:
The 17s vs. 11s speed difference is not worth making this harder to maintain, and adding the potential for breaking or ignoring errors as we change the schema. This isn't a real-time service so care much more about consistency, correctness, and maintainability than performance.

To get to your one switch at the top, how about:
```
from vaccine_feed_ingest_schema.common import BaseModel

if optimized:
   BaseModel.create = BaseMode.construct
else:
  BaseModel.create = BaseMode.__init__
```

And then you can use the `create` method everywhere e.g. `Contact.create(phone=num)`

Reply:
Done.